### PR TITLE
add example config with Postgres

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+==== Example configuration with PostgreSQL
+It is recommanded to use a PostgreSQL database in a ProActive prod environment.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:787]: Configuring ProActive with an external database such as PostgreSQL is recommended in a production environment.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+In the following, we show an example of PostgreSQL installation and configuration to be used along with ProActive.
+
+===== PostgreSQL version
+PostgreSQL version 12.2 or 12.9 is required to be installed. PostgreSQL could be installed on the same machine as for ProActive or on a separate server (on-prem or on the cloud).
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:791]: PostgreSQL versions 12.0 to 12.11 are fully compatible with ProActive. More recent versions of PostgreSQL have not been fully tested yet for compatibility. 
PostgreSQL server can be installed on the same machine as the ProActive server or on a separate machine.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+===== Initialize the database
+To initialize the DB, some SQL queries should be executed in the PostgreSQL server machine.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:794]: To initialize the database, ...

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+To do so, please create the init script in the /tmp folder for example and put the following SQL queries:
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:795]: In order to do so, the following SQL queries can be defined in a SQL initialization script:

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 5]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+....
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:796]: add an empty line before, and use the source with syntax highlighting syntax

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 6]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+GRANT USAGE ON SCHEMA public TO pca;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO pca;
+GRANT CONNECT ON DATABASE notification TO notification;
+GRANT USAGE ON SCHEMA public TO notification;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO notification;
+
+ALTER SYSTEM SET max_connections ='500';
+....
+
+Set the host name or IP address(es) to listen to:
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:831]: When the PostgreSQL server is installed on a separate machine, the `listen-addresses` setting should be modified to allow remote connections:

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 5 above)
Comment (fviale) [reply src/docs/admin/ProActiveAdminGuide.adoc:796]: add also a title to the file `init.sql`

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 7]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+....
+sudo vim /opt/postgres/data/postgresql.conf
+listen-addresses = ‘IP_ADDRESS’
+....
+
+Then, execute the script
+....
+psql -f /tmp/init.sql
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:839]: psql -f init.sql

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 8]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+ALTER SYSTEM SET max_connections ='500';
+....
+
+Set the host name or IP address(es) to listen to:
+....
+sudo vim /opt/postgres/data/postgresql.conf
+listen-addresses = ‘IP_ADDRESS’
+....
+
+Then, execute the script
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:837]: Execute the initialization script inside the PotsgreSQL database:

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 9]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+....
+psql -f /tmp/init.sql
+....
+
+===== Prepare the database purge operation
+In normal PostgreSQL operation, tuples that are deleted or obsoleted by an update are not physically removed from their table;
+they remain present until a purge (VACUUM) is done. Therefore it is necessary to do a purge periodically, especially on frequently-updated tables.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:844]: Additionally, a vacuum operation does not by default remove orphaned Large Objects from the database. Identification and removal of these objects is performed by the `vacuumlo` command.

As the ProActive database contains many large objetcs, it is essential to apply both `vacuumlo` and `vacuum` operations in order to control disk space usage over time.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 10]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+In our case, the preparation for the DB purge operation is done in two steps.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:845]: Prior to execute the above operations, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 11]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+In the first step, we alter several tables in the databases after the first start of the server by executing the following SQL queries:
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:847]: After the first start of the ProActive server, the following SQL queries will alter several tables of the database (this is needed due to a PotsgreSQL driver issue):

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 12]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+....
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:848]: use source code with syntax highlighting, you should also name the file

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 13]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+alter table rule_error alter column error_message TYPE oid USING error_message::oid;
+alter table submittedrule alter column failure_message TYPE oid USING failure_message::oid;
+\q
+
+# PCA database
+\c pca
+alter table variable alter column name TYPE oid USING name::oid;
+alter table variable alter column value TYPE oid USING value::oid;
+\q
+....
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:886]: Remember to add an empty line before and after source code blocks

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 14]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+In the second step, we remove large objects stored in these tables by executing the following SQL queries.
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:887]: After the initial configuration step is done, it is possible to execute a full vacuum by using the following commands:

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 15]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+# pca database
+vacuumlo -v pca
+vacuumdb -f -v -d pca -t pg_largeobject
+
+# check autovacuum
+SHOW autovacuum;
+....
+
+**Notes:**
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:905]: you can use the asciidoc admonition syntax `NOTE:`

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 16]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+ * The command vacuumdb is executed implicitly if AutoVacuum is enabled.
+ * The command vacuumlo consumes a lot of RAM and disk space depending on the number of large objects to be removed.
+
+
+===== Download the Postgres driver
+The Postgres driver have to be downloaded and added to the addons folder of ProActive (SCHEDULER_PATH/addons/)
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:912]: ... driver has to be ... and added inside the addons ...

these 2 sections should be before "Prepare the database purge operation"

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 17]
```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+The link to download a Postgres driver version compatible with Postgres 12.X is: `https://jdbc.postgresql.org/download/postgresql-42.2.11.jar`
+
+===== Configure ProActive
+To configure ProActive to connect to the Postgres DB, we need to edit the database configuration files as follows:
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc:917]: as follows.

The database passwords should be modified according to your chosen passwords. Note that passwords can be encrypted in configuration files as explained in section `<<_encrypting_configuration_passwords>>`

**Review by fviale — APPROVED** [review]

**Review by amouhoub — APPROVED** [review]

**Pull request merged**